readme.md

```diff
@@ -109,6 +109,8 @@ Are you a Hackbright alumna or student?
 ## Educational
 
 - [Idiomatic Python 3](https://github.com/jerry-git/learn-python3#idiomatic-python)
+- [Problem Solving with Algorithms and Data Structures using Python](https://interactivepython.org/runestone/static/pythonds/index.html)
+
```

Comment:
Delete line 113 to match whitespace of rest of document

Reply:
The extra line has been deleted.